vars/withPod.groovy

```diff
@@ -7,14 +7,13 @@ import static org.centos.contra.Infra.Defaults.*
  * @param body
  * @return
  */
-def call(Map config=[:], Closure body){
-
+def call(Map config=[:], Closure body) {
     def infraUtils = new Utils()
 
     env.userPodName = config.podName ?: "pod-${UUID.randomUUID()}"
 
-    config.openshift_service_account = config.openshift_service_account ?: openshiftServiceAccount
-    config.openshift_namespace = config.openshift_namespace ?: infraUtils.getOpenshiftNamespace()
+    String openshiftServiceAccount = config.openshift_service_account ?: openshiftServiceAccount
+    openshiftNamespace = config.openshift_namespace ?: infraUtils.getOpenshiftNamespace()
```

Comment:
This should be:
`def openshiftNamespace = config.openshift_namespace ?: infraUtils.getOpenshiftNamespace()`
or (preferably)
`String openshiftNamespace = config.openshift_namespace ?: infraUtils.getOpenshiftNamespace()`

Reply:
for similar reasons as above regarding variable definition